src/App.js

```diff
@@ -6,19 +6,26 @@ import PetForm from './pages/PetForm'
 import PetProfile from './pages/PetProfile'
 import TaskForm from './pages/TaskForm'
 import { loadFromLocal, saveToLocal } from './services'
-import Tasks from './tasks.json'
 import Pets from './pets.json'
 
 export default function App() {
-  const [tasks, setTasks] = useState(loadFromLocal('tasks') || Tasks)
-  useEffect(() => {
-    saveToLocal('tasks', tasks)
-  }, [tasks])
-
   const [pets, setPets] = useState(loadFromLocal('pets') || Pets)
   useEffect(() => {
     saveToLocal('pets', pets)
   }, [pets])
+
+  function addTask(task) {
```

Comment:
Ich verstehe auf den ersten Blick, was das alles für const sind, die du in der addtask function definierst. Müssen die da wirklich alle rein?

Reply:
ich glaube schon, da ich auf so viele Sachen zugreifen muss. Erst das Tier, dann task, dann checkbox. Ich frage die Coaches, ob das einfacher geht :)